analyses/2019_03_agarwalishita_analysis.md

```diff
@@ -0,0 +1,30 @@
+The major difference between the dataset required to train/synthesize ad blocker and data blocker is described in this piece of text.
+
+**Ad blocker:**
+Its data mainly consists of filter rules which comes from EasyList, EasyPrivacy, etc. which consist of known ads on English Language websites all over the world. You can check the whole list [here](https://easylist.to/easylist/easylist.txt).
+
+  Ad blocker blocks data based on two things:
+
+1.  *Url domains*: Based on the dataset, ad blockers know which domains are not part of ad networks and those domains calls are blocked. You might have gotten idea about this idea after looking at the EasyList.
```

Comment:
What do you mean by dataset here - the block list itself?  I think talking about how block lists are generated is important.

> You might have gotten idea about this idea after looking at the EasyList.

I would suggest something slightly more formal such as "The EasyList block list is open source and can be reviewed [here](https://easylist.to/easylist/easylist.txt)"